### PR TITLE
Configure missing tracing attributes for kafka, amqp rabbitmq.

### PR DESCRIPTION
**Pull request by ozangunalp**
Restore system attributes from connector name to messaging system name

@radcortez I noticed that messaging.system attribute where changed, please review these fixes.

### CODE REVIEW

File: smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java [hunk 1]
```diff
@@ -39,7 +38,7 @@ private static class AmqpMessagingAttributesGetter implements MessagingAttribute
         // Required
         @Override
         public String system(final AmqpMessage<?> amqpMessage) {
-            return AmqpConnector.CONNECTOR_NAME;
+            return "AMQP 1.0";
```
Comment (radcortez) [inline smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java:41]: I am wondering if this should be `amqp` only?

**Review by radcortez — APPROVED** [review]

File: smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java (hunk 1 above)
Comment (ozangunalp) [reply smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java:41]: I hesitated on that as well, then kept the one we had before.

Comment (radcortez) [reply smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java:41]: Ok, just leave that as is :)

Comment (cescoffier) [reply smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java:41]: AMQP is misleading. For most people it's the AMQP from RabbitMQ (8. something), and not AMQP 1.0. So, better be clear about that.

Comment (radcortez) [reply smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/tracing/AmqpAttributesExtractor.java:41]: We can set the system name and the protocol. For RabbitMQ we had `amqp` set as the protocol, but we cleared that out.

For AMQP, I guess that both the system and protocol are `amqp`.

**Pull request merged**